unlock-app/src/hooks/useSymbol.ts

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from '@tanstack/react-query'
+import networks from '@unlock-protocol/networks'
+import { useWeb3Service } from '~/utils/withWeb3Service'
+
+interface GetLockSymbolProps {
+  lockAddress: string
+  network: number
+  contractAddress?: string | null
+}
+export const useGetLockSymbol = ({
+  lockAddress,
+  network,
+  contractAddress,
+}: GetLockSymbolProps) => {
+  const web3service = useWeb3Service()
+  const baseCurrencySymbol = networks?.[network].nativeCurrency.symbol
+
+  return useQuery(
+    ['getLockSymbol', lockAddress, network, contractAddress],
+    async () => {
+      if (contractAddress) {
+        const symbol = await web3service.getTokenSymbol(
+          contractAddress,
+          network
+        )
+        return baseCurrencySymbol || symbol
+      }
+
+      return baseCurrencySymbol
+    }
+  )
```

Comment:
seems redundant? We should use hook that gets symbol from subgraph or create an alternative which we can pass fields to.

Reply:
`symbol` from the subgraph does not match the `symbol` we need here, also the code above we using in a different place and we can replace it with this hook